packages/commands/src/index.ts

```diff
@@ -1187,6 +1229,18 @@ export namespace CommandRegistry {
       if (parts.cmd) {
         mods += '\u2318 ';
       }
+      if (parts.arrowLeft) {
+        mods += '\u2190 ';
+      }
+      if (parts.arrowUp) {
+        mods += '\u2191 ';
+      }
+      if (parts.arrowRight) {
+        mods += '\u2192 ';
+      }
+      if (parts.arrowDown) {
+        mods += '\u2193 ';
+      }
```

Comment:
This was only added to the Mac condition, this means that as-is the arrows are not displayed on other operating systems at all. I will fix that.

Reply:
See 526b1438bec0848a5ca91c6458e1e2390b805c99.